MH4U Dex/MonsterDetailsViewController.m

```diff
@@ -29,39 +34,61 @@ @implementation MonsterDetailsViewController
 - (void)viewDidLoad
 {
     [super viewDidLoad];
-    [self.lowRankSubView setHidden:YES];
+    self.overviewSubView.hidden = NO;
+    self.lowRankSubView.hidden = YES;
+    self.highRankSubView.hidden = YES;
+    self.GRankSubView.hidden = YES;
```

Comment:
Have an `IBOutlet` pointed at the segmented control and pad that property as the sender.